pkgs/os-specific/linux/realmd/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv
+, autoreconfHook
+, docbook_xml_dtd_412
+, docbook_xml_dtd_42
+, docbook_xml_dtd_43
+, docbook_xsl
+, fetchFromGitLab
+, glib
+, intltool
+, libkrb5
+, libtool
+, libxslt
+, openldap
+, pkg-config
+, polkit
+, systemd
+, xmlto
+}:
+# doc-related (--disable-doc): docbook_xml_dtd_412 docbook_xml_dtd_42 docbook_xml_dtd_43 docbook_xsl libxslt xmlto
```

Comment:
I am not sure what that comment is supposed to tell me

Reply:
This was basically just WIP, as building the docs pulled in a huge amount of dependencies and I thought about disabling docs later using `--disable-doc`.